ethereum/contracts/interfaces/relayer/IWormholeReceiver.sol

```diff
@@ -1,48 +1,67 @@
-     *
-     * @param deliveryData - This struct contains information about the delivery which is being performed
-     * @param signedVaas - Additional VAAs which were requested to be included in this delivery. They are guaranteed to all be included and in the same order as was specified in the delivery request.
-     * NOTE: These signedVaas are NOT verified by the Wormhole core contract prior to being provided to this call.
-     * Always make sure parseAndVerify is called on the Wormhole core contract before trusting the content of a raw VAA,
-     * otherwise the VAA may be invalid or malicious.
-     */
-    function receiveWormholeMessages(
-        DeliveryData memory deliveryData,
-        bytes[] memory signedVaas
-    ) external payable;
+/**
+ * @notice DeliveryData is the struct that the relay provider passes into 'deliver' containing an
+ *     array of the signed wormhole messages that are to be relayed
+ *
+ * @custom:member sourceAddress - the (wormhole format) address on the sending chain which requested
+ *     this delivery. Any address is able to initiate a delivery to anywhere else.
+ * @custom:member sourceChainId - the wormhole chain ID where this delivery was requested.
+ * @custom:member maximumRefund - the maximum refund that can possibly be awarded at the end of this
+ *     delivery, assuming no gas is used by receiveWormholeMessages.
+ * @custom:member deliveryHash - the VAA hash of the deliveryVAA. If you do not want to potentially
+ *     process this delivery multiple times, you should store this hash in state for replay
+ *     protection.
+ * @custom:member payload - an optional arbitrary message which was included in the delivery by the
+ *     requester.
+ */
+struct DeliveryData {
+  bytes32 sourceAddress;
+  uint16 sourceChainId;
+  uint256 maximumRefund;
+  bytes32 deliveryHash;
+  bytes payload;
+}
 
-    /**
-     * @notice DeliveryData - This struct contains information about the delivery which is being performed.
-     *
-     * @custom:member sourceAddress - the (wormhole format) address on the sending chain which requested this delivery. Any address is able to initiate a delivery to anywhere else.
-     * @custom:member sourceChain - the wormhole chain ID where this delivery was requested.
-     * @custom:member maximumRefund - the maximum refund that can possibly be awarded at the end of this delivery, assuming no gas is used by receiveWormholeMessages.
-     * @custom:member deliveryHash - the VAA hash of the deliveryVAA.
-     * @custom:member payload - an arbitrary message which was included in the delivery by the requester.
-     */
-    struct DeliveryData {
-        bytes32 sourceAddress;
-        uint16 sourceChain;
-        uint256 maximumRefund;
-        bytes32 deliveryHash;
-        bytes payload;
-    }
+interface IWormholeReceiver {
+  /**
```

Comment:
I think most of the diff in this file is actually not intended, as I think you started this PR before commit d560a95e08c08c774fbbd2e69cc9c1e811851313 (which is reflected in the old code but not here). Probably the main diff you intended here is removing the struct out of the interface?

Reply:
Just another huge diff due to code formatting... (I recall that some comments had 2 spaces between the asterisk and the text and other comments only had one and I fixed that - not sure if this was here).

I originally branched before commit [d560a95](https://github.com/wormhole-foundation/wormhole/commit/d560a95e08c08c774fbbd2e69cc9c1e811851313) but then did a caveman rebase (couldn't get my actual branch to properly rebase on the latest upstream and so I just cloned the entire thing again and submitted all changes as a single commit there...).